Post/models.py

```diff
@@ -1,8 +1,7 @@
 from django.db import models
 
 
-class post(models.Model):
-
+class Post(models.Model):
```

Comment:
This change affects other files, please make sure that you replace  'post' with 'Post' in those files.
for example- in the file '0002_test_data.py' there is 'post' instead of 'Post'..